### DISCUSSION TITLE
很遗憾，NanUI 将暂停开发。

### DISCUSSION THREAD

**Discussion by XuanchenLin** [XuanchenLin/NanUI#367] [公告 / Announcements]
做出此艰难决定本人只能深表抱歉。

由于 NanUI 项目一直以来都比较小众，因此社区生态并没有建立起来，至今为止几乎没有收到过任何 PR，这也造成项目目前还是由我一个人开发这种局面。

NanUI 项目目前在行业内已经有一定的用户基础，也不乏有像中国联通、百旺金赋、城市快捷酒店、申通等这些有点名头的用户，但目前项目依然采取开源免费的政策，之前偶尔会有朋友通过二维码5块10块的进行捐助，另外靠着提供支持 H264 编码的 CEF 编译库“逼捐”每年也最多有一两千的收入，但这对一个项目的维持和开发是远远不够的。

非常不幸的是，本人所在公司也因疫情的原因去年（2022年）初彻底宣布裁减所有开发岗位，因此本人也只能顺应大流在36岁这个尴尬的年纪失业，原本在上班的时候可以靠着闲暇摸鱼时间开发和维护 NanUI，但这一失业，这种机会也完全丧失了。

失业后，经过深思熟虑我只能跨行业以销售钢材为生，写代码从曾经的职业变成了业余兴趣爱好，毕竟得生活，得吃饭，我也只能先解决好温饱问题。

经过一年的跨行业工作，俗话说隔行如隔山，本人只能耗费大量精力从0学起，加之这两年经济形势糟糕，即使消耗了我无数的时间和精力我还是很难在业绩上有所突破。为了生存，本人只能花费更多的时间和精力去谈单，去销售，去收款，因此已经很难再腾出时间来开发和维护 NanUI 项目，对此我深感无奈，也希望后面生活和工作稳定后能腾出时间来继续维护 NanUI。

目前 NanUI 的最新版本已经开发完成，本来打算把这个版本作为 NanUI 的 1.0 正式版进行发布，但是实在没有时间和精力去编写文档和制作示例程序，因此这个版本的发布只能延期，等后面有时间把文档和示例编写完成后再进行发布。只能幸苦各位继续使用目前的 0.9.109 版本的 NanUI 了。

当前 0.9.109 版本的 NanUI 在功能上已基本稳定，也没有什么致命的 Bug，因此可以用于开发各种生产环境的软件。如果遇到问题，可以到 QQ 群咨询已经开发有成熟产品的各位大佬，也可以私聊我留言，我有空的时候尽量回复。

感谢各位多年来会 NanUI 项目的支持和关注，后会有期！

此致，

林选臣 于 2023年国庆。

**Comment by woshidama323** [comment]
哎，太难了，是不是win的时代要过去了

**Reply by XuanchenLin** [reply]
这跟windows没关系，主要是近几年国内的经济不行，本来软件就属于三产，经济不行没钱谁还掏钱做软件😂

**Reply by XiaoNianPoPo** [reply]
你这搞得我这17岁还在实习的中专很慌啊

**Reply by huster-songtao** [reply]
WIN的时代没过去啊，我还在开发桌面软件，行业需求一直存在的

**Reply by 532777032** [reply]
需求一直有啊，但是确实小众了

**Reply by hfkai** [reply]
@XiaoNianPoPo 中专？大专都找不到工作了，还中专

**Reply by whevether** [reply]
中专你还是趁早放弃吧。  现在啥岗位都看学历。  大专都没几个岗位。  中专更是实习了估计连面试都没有

**Reply by XiaoNianPoPo** [reply]
我找到了一个PHP的，不过月薪3K，还行吧经理人挺好的，就是工资很低，但我都一个中专了，就也不计较了，虽然又干前端又干后端。。。。。。

**Reply by XiaoNianPoPo** [reply]
我是想大专之后专插本的，广东这边也还行吧

**Reply by skypesky** [reply]
> 我找到了一个PHP的，不过月薪3K，还行吧经理人挺好的，就是工资很低，但我都一个中专了，就也不计较了，虽然又干前端又干后端。。。。。。

现阶段能够学习到各种技术,增强实力,感觉还是不错的.之后再考虑去广州,深圳发展也是不错的选择.

**Comment by anghunk** [comment]
中年人的程序员生涯这么艰难吗。。。唉，在想要不要提前找好退路。

**Reply by 532777032** [reply]
听我说，35岁不是用脑的极限，是身体的极限，我没啥基础病，35岁都集中爆发了，早想好后路吧，

**Reply by huster-songtao** [reply]
身体应该还好，最主要是国情对大龄码农非常不友好，要是在香港或者新西兰，可以或者很湿润

**Reply by rustdesk** [reply]
@532777032 你衰的早

**Reply by idragonet** [reply]
35还好，关键是40.45.

**Reply by 532777032** [reply]
@rustdesk 别这样老铁，-_-!!

**Reply by Amos-li** [reply]
极限在你的心!

**Reply by Amos-li** [reply]
40+ 的一线飘过

**Comment by ypeng-ming** [comment]
致敬！感谢您做的一切，愿您一切顺利，转了20块聊表敬意。
另外我觉得像您这么优秀得程序员就算36岁了也肯定可以找个一个还行的开发相关的工作的，为什么你选择做钢材销售呢？是因为销售的职业生涯不会被年龄影响，反而会越来越吃香吗？

**Reply by duobei** [reply]
可能是家里有点门路吧，国内的远程岗位又少，这些只是臆测

**Reply by ypeng-ming** [reply]
不太可能吧 有门道应该不会做销售 除非是为了锻炼自己的相关能力 看看林大哥怎么说吧

**Reply by U97F3** [reply]
我认为应该是有货源的门路

**Reply by dcsite** [reply]
36 岁就算能找到养家户口的工作，46 岁呢？很多人都是从 26 岁觉得现在工作能吃饱，温水煮青蛙到 36 岁的

**Reply by hfkai** [reply]
很难找到跟之前一样工资的工作了

**Reply by punkmonday** [reply]
谦虚说是销售，其实是创业

**Comment by drmingdrmer** [comment]
祝好, 江湖再见

**Comment by ironfeet** [comment]
作为曾经的 .Net 和 C# 忠实码农，看到此文真是感慨万千。祝一切顺利

**Comment by dogeow** [comment]
想贡献的人弄不了，弄得了的人不屑贡献，这就死循环了。

**Reply by rustdesk** [reply]
基本说到点上

**Reply by jhjade** [reply]
我都转行3年多了，目前前端研发

**Reply by yichengxian** [reply]
那就相当于没转一样得

**Reply by whevether** [reply]
你这转个寂寞。这是转行吗.

**Reply by leaf3woods** [reply]
在文档不全的情况下，提什么pr哦，熟悉项目都费劲

**Reply by punkmonday** [reply]
你可以提写文档的pr

**Comment by jtyjty99999** [comment]
50块聊表心意，祝好 祝早日工作稳定

**Comment by tangjianke** [comment]
祝好，同是35岁大龄程序员

**Comment by jonpostel** [comment]
刚看到新闻，我早在朋友圈说过，开源养活不了自己。祝你好运。

**Reply by huster-songtao** [reply]
承德某码农3年赚了一百多万，被zf没收了

**Reply by jacob-07** [reply]
很好奇，什么途径接国外私活3年能赚那么多

**Reply by punkmonday** [reply]
应该是btc支付，现在国内有少部分可以使用btc支付的行业

**Reply by AndyFightting** [reply]
上github的各位都得当心了

**Comment by Rahtasim** [comment]
经济下行阶段大家都不好过，这一年来好多开源项目作者坚持不下去了。

哎……同样作为大龄码农，我还是挺喜欢敲代码的，但是单纯的喜欢并不能自动地转化成面包。

大哥要保重身体呀，共勉 ❤️

**Comment by XuanchenLin** [comment]
感谢各位的支持，今天在开车在去催款的路上一直都有不停的收到各位的捐赠和祝福，非常感动！

这几年确实是因为经济形式不好，去年一直在给本地的某国企供货，几乎可以说倾尽家产垫到工程里，然而一年过去了，一分钱都没有收回来。一直都有房贷需要还，之前预计今年年初怎么都能回点款，可惜只怪自己太年轻，工程垫款这块水太深，导致今年到下半年之后生活费都TM成问题了，所以只有暂时把精力放在收款上，要不弄不好再过几个月我也成失信人员了，哈哈。

所以只能先跟各位关注和喜爱 NanUI 项目的朋友们说句抱歉了，其实目前1.0版本主体已经开发完毕，还有些细节需要调整和修改，另外就是文档和示例也暂时没有时间精力去写，只能后面收款的事情有进展了再抽空完成剩下的工作。

对于未来 NanUI 项目的发展，我想可能会效仿其他一些国内开源项目采取基础版继续使用MIT开源免费，进阶版收一定费用的方式来为项目的开发工作募集一些资金贴补家用。我的想法是进阶版只是提供额外的工具类方便开发，并不会缩减开源版目前的功能和特性，有需求或者是想对项目进行支持就付费解锁进阶功能即可。

我从10岁开始玩BASIC，到后来的Turbo C，Pascal，大学读了计科开始玩C#到现在，感觉从学校出来没有几年，36岁难道就特么毕业了？怎么想都想不通，真是无奈至极～

**Reply by ypeng-ming** [reply]
这操蛋的社会

**Reply by itfanr** [reply]
为啥不找份c#岗位继续工作呢？

**Reply by LAnubis** [reply]
36的c#程序员了，感觉现在的这份工作也是最后一份写代码的工作了，如果转的话，如果还是写代码的话估计只能上位机了，大龄程序员在二三线城市还好点，没一线城市那么的无奈，年纪大了不想写代码了，也不知道应该做啥了啊

**Reply by 532777032** [reply]
我35了还在一线前端，点开我主页，你看看我在玩啥。能给大家一点灵感

**Reply by XuanchenLin** [reply]
全部钱垫在工程项目上，不收回来哪里有心思去上班😂

**Reply by geqiaoyu** [reply]
建议去工厂IT部门的，或者上位机开发，c#还是吃香的，作为同年人跟你一样，最后还是去了甲方工厂，无奈啊

**Reply by idragonet** [reply]
唉!我和一样，贪心了。 搞一个东西款收不回了，等亲戚卖房还款给我。

**Reply by LAnubis** [reply]
不想弄上位机，但是想去厂里  哈啊哈

**Reply by gitzjp** [reply]
好奇问一下大佬。为啥都做微软他们家的啊

**Reply by hellohejinyu** [reply]
> 好奇问一下大佬。为啥都做微软他们家的啊

作者上面说了大学开始就一直搞 C# 了，C# 就是微软的呀。

**Reply by huster-songtao** [reply]
我还在做上位机，上位机收入还行吧，几千到几万都有，有行业门槛的工资一般都比较高，一般人还做不了

**Comment by tomieric** [comment]
下岗工人来报到

**Reply by 532777032** [reply]
35还在一线的来报到了

**Comment by edo-zhou** [comment]
c#写这么6 不去新西兰等国外的国家试试吗

**Reply by ypeng-ming** [reply]
拖家带口的 不好润

**Reply by huster-songtao** [reply]
好润，我哥们就去新西兰了，他81年的，现在在奥克兰，

**Reply by idragonet** [reply]
分享下细节。

**Reply by jacob-07** [reply]
35岁，想润

**Reply by tonyqus** [reply]
新西兰工作并不好找，更适合有钱人，中产慎入。

**Comment by 532777032** [comment]
要不跟我学微信群聊机器人吧，我35了 最近也慌，就试试自媒体，到时候机器人做完在集成下GPT进去，应该挺有商业价值的[用node开发微信群聊机器人第①章](https://mp.weixin.qq.com/s/fkUyfxiDMyzVFc2qrWn37A)

**Reply by huster-songtao** [reply]
合规是个问题，小众市场

**Reply by 532777032** [reply]
事在人为啊，不是说一定用来对接gpt，只是举例，可以做的很多

**Reply by iron2han** [reply]
易拉罐414

**Reply by hellohejinyu** [reply]
能不能别蹭热度打广告了，像话嘛。

**Comment by r4ntix** [comment]
祝好，希望一切顺利！💪

**Comment by huster-songtao** [comment]
> 目前 NanUI 的最新版本已经开发完成，本来打算把这个版本作为 NanUI 的 1.0 正式版进行发布，但是实在没有时间和精力去编写文档和制作示例程序，因此这个版本的发布只能延期，等后面有时间把文档和示例编写完成后再进行发布。只能幸苦各位继续使用目前的 0.9.109 版本的 NanUI 了。

我可以帮忙编写文档和制作示例程序，我78年的老码农，现在还在写代码。

**Comment by vittorezhang** [comment]
祝好，一切顺利~

**Comment by devillove084** [comment]
祝好，一切顺利。

**Comment by Deja-vuuu** [comment]
**_而你，我的朋友，你是真正的英雄 ！_**   
                                                            -致敬

**Comment by WenQizhaNg1** [comment]
祝好

**Comment by shenadmin** [comment]
国内的工作情况就是这样的,大龄背着房贷,上有老下有下 小的,离35也不远了

**Comment by luanxuechao** [comment]
祝好运。

**Comment by lekai63** [comment]
钢材仅说线下销售的化，利润太薄太薄了，甚至还倒挂、贴钱。卖钢材还得期现结合，依靠区域差、品牌差等来卖。楼主好运，钢材也很难做（尤其是房产不景气后，需求相对减少）

**Comment by xc2f** [comment]
我也要辞去8年前端开发职业生涯，转行做建筑了。祝好。

**Reply by idragonet** [reply]
我也是NET，40出头了（本人今年有2个机会，因为薪资不增加所以没有选择），我不太相信大佬36找不到开发的工作？ 是不是自己对开发厌倦了？

**Reply by xc2f** [reply]
目前呆的单位氛围不错，但是从疫情后开始业务不行，再呆10年也是混吃等死。
这一行干久了感觉自己待人接物有问题，找个和人打交道的行业闯荡一番。

**Reply by huster-songtao** [reply]
我45岁，在开发.NET MAUI，REACT/VUE，嵌入式，后台，全栈开发

**Reply by wangweiying303** [reply]
现在建筑刚也也感觉不太景气哎。。。我是学建筑的现在写写代码

**Reply by lee576** [reply]
这个年龄，在职感受不到，你尝试下跳槽，看难度有多大

**Reply by rustdesk** [reply]
咋都换着坑跳

**Reply by 532777032** [reply]
这个时间跳写代码，风险太大了。@wangweiying303

**Reply by lizhancheng** [reply]
我也是辞去了8年前端，现在去国企做信息化管理了

**Comment by llussy** [comment]
祝好，希望一切顺利！

**Comment by delonglimin** [comment]
我也准备看卖钢材的职业，在聊城，楼主可以留个联系方式，或者加个v：delong123abc

**Comment by Flithor** [comment]
微博观光团

**Comment by funnyrabbit87** [comment]
虽然没用过，也不是做C＃的，但是和LZ一模一样，36，裁员失业。LZ还能去卖钢材，我是一点出路都没有。共勉共勉啦。

**Comment by wjdq** [comment]
加油

**Comment by TheoBlake** [comment]
祝好~但是一身技术本领有些可惜了，大佬可以考虑考虑做出海saas产品

**Comment by zouHuaLu** [comment]
也算是给我们后来者敲响警钟了，要未雨绸缪，居安思危

**Comment by marlonLau** [comment]
.net开发，已经失业3个月了。。祝好

**Reply by whevether** [reply]
今年搞it的岗位锐减。  我朋友之前做外贸erp的。公司从巅峰400多人倒现在100人不到。开发人员从近200号人倒现在30号人。 因为外贸订单锐减。

**Reply by huster-songtao** [reply]
我这边有点小活，医疗方向，做软硬件对接，硬件部分我已经弄好了，设备太多了，我一个人忙不过来，想找人帮忙，有没有兴趣？有兴趣加我微信MercadoCaribe

**Reply by huster-songtao** [reply]
我对学历没有歧视，能干活就行。我这边开发技术栈是.NET MAUI，React，C#，JavaScript这些，后面还有一个WebRDP远程桌面和音视频通话的活要干，主要也是拿开源的改改交付给客户。

**Reply by lee576** [reply]
我2023年9月失业的，也在找活干

**Comment by wc136706639** [comment]
全体起立致敬

**Comment by rustdesk** [comment]
开源可以养活自己，35/40也不是问题，只是你要找对方向

**Reply by idragonet** [reply]
找对方向是难题。

**Reply by rustdesk** [reply]
想要站着赚钱，当然难了

**Reply by snowman** [reply]
那怎么躺着赚钱

**Reply by lizhancheng** [reply]
要是能有大佬开发远程桌面这般能力都不成问题

**Reply by huster-songtao** [reply]
@lizhancheng 我就在开发远程桌面啊，你可以搜Remotely，我就基于这个做二开

**Reply by Jason8996** [reply]
方向也是会随着时间改变的，有几个项目能向Linux内核，GNU编译器以及ffmpeg等这样的基础设施以及工具库一样长久？况且这样的项目也不是靠个人去维持的

**Comment by zhang-wei-jian** [comment]
前程似锦，祝大家好运早日找到自己真正喜欢的赚钱的方向

**Comment by dswfort** [comment]
18年的一个项目有幸用过一次，还是0.6版，看到这个公告好有感触

**Comment by TerryZ** [comment]
真是令人伤感

**Comment by laosanyuan** [comment]
祝大佬有个好收获，能早点回来继续搞开发。这个行业太难了

**Comment by reatang** [comment]
不说35了。30岁各种小疾就无缘无故的来了。

**Comment by Wonderchn** [comment]
怎么突然就变成这样呢

**Comment by Nicolasking007** [comment]
诸君，共勉

**Comment by YouXianMing** [comment]
真是令人伤心😭

**Comment by Lruihao** [comment]
共勉

**Comment by iambus** [comment]
虽然不了解这个项目，但是看你的中文运用，短短数百字，组织清晰，用词严谨，相信你也是一位十分优秀的开发者。惜别。

**Comment by outthin** [comment]
哎，共勉吧，虽然这个行业，没让我大富大贵，但是他支撑了我的温饱。

**Comment by xincheng213618** [comment]
诸君，共勉

**Comment by cisiqo** [comment]
35岁刚被优化，找工作中。学历不好，难

**Comment by qjaden** [comment]
建议趁着这几年窗口期润出去，后面计划经济就难出去了

**Reply by soyadokio** [reply]
请问“窗口期”怎么理解？为什么后面会难？
单纯不理解这个问题，忘指点。

**Comment by wangqi** [comment]
国内的环境缺少福利兜底，导致有能力的人还要为温饱发愁，太可惜了。如果有机会的话，润到海外来吧
加油，祝早日收回欠款。

**Comment by owlcoocoo** [comment]
虽然我也是 .net 开发者，但现在在看其他机会。
祝好，共勉。

**Comment by c-poet** [comment]
国内环境真让人担忧，25岁已经开始瑟瑟发抖！！！

**Comment by asd0109** [comment]
感谢你所作出的贡献，祝你在未来的工作和生活中一切顺利。

**Comment by uscj** [comment]
作为大龄程序员之一，对于写代码并没有太大热情，只是为了生存，去年和今年连续被裁员两次，虽然现在仍在写代码，但是薪资无法与之前相提并论，也是非常无奈，过两年就35岁了，如果再失业，计划着回家乡搞民宿。共勉

**Reply by whevether** [reply]
兄弟我家就是搞民宿的。  民宿跟着旅游周期的。 旅游淡季的时候基本没钱入账的。

**Reply by uscj** [reply]
大环境非常差，说不定哪天就失业了，在这个非常尴尬的年龄，找工作又非常难。听我同学说过，现在搞民宿竞争也挺大的，难啊

**Comment by AlphaYu** [comment]
唉。。

**Comment by NeverMorewd** [comment]
致敬每一个为开源社区做贡献的人。

**Comment by world56** [comment]
上了30岁，年龄危机5年来一次，尽早尽可能降低负债吧，相对靠谱的一条路。

**Comment by fqdeng** [comment]
祝好，希望能早日拿回款项

**Comment by webzhouyan** [comment]
**加油💪🏻**

**Comment by llc3s** [comment]
祝好

**Comment by languoguoqaz** [comment]
加油💪🏻

**Comment by transfercai** [comment]
千言万语说不出口，汇成加油两字，共勉

**Comment by ccnetcore** [comment]
共勉

**Comment by StormHub** [comment]
我在澳洲， 想帮你维护。 @XuanchenLin  有空联系我
外面一般年纪越大， 越是吃香。 国内怎么这样搞？？？？
Unbelievable !!

**Reply by reatang** [reply]
亚服老玩家不如狗

**Reply by jacob-07** [reply]
请问大佬怎么去的澳洲，我35了，也想试一下

**Reply by StormHub** [reply]
我在澳洲上的学

**Reply by jacob-07** [reply]
方便留一下邮箱吗？交流一下

**Reply by tonyqus** [reply]
国外呆久了，真的很难理解国内的情况。35岁在大中型厂被搞很正常，比如圈内某张姓大佬，就是因为35岁被腾讯毕业的。

这也导致了国内的技术圈很难再出资深架构师了，因为很多人是35岁以后开窍的，那种天资聪慧，35岁之前混出样的毕竟是少数。

**Comment by purecafe2019** [comment]
方便留个联系方式吗？

**Comment by isMrFan** [comment]
马上进入35的圈子,担心被裁员 还有一屁股外债真是很难啊，同时有没有前端开发一起维护我这个小UI啊刚做

**Reply by whevether** [reply]
哥们我劝你放弃。这种很多。如果run不出去。与其浪费精力时间在这种烂大街技术上。还不如早早想好其他出路

**Reply by isMrFan** [reply]
谢谢你的忠告,但是我靠这玩意能P8升职用薪资好搞个50+，等被裁员的时候我就能财务小自由了,本人做GTP，AI制图,游戏创作,抖音卖货都在搞

**Reply by isMrFan** [reply]
老哥还有别的好的出路吗？给我给我建议我取取经

**Reply by Deja-vuuu** [reply]
你都p8了还缺钱么？

**Reply by isMrFan** [reply]
缺啊老哥,结婚，父母，我和对象都是独生子女，对象幼师工资不高花销老大了，房子还没买呢

**Reply by isMrFan** [reply]
还没8啊就是写点东西好升级答辩到8啊这样就不用裁员了啊，现在是7

**Reply by tonyqus** [reply]
人家找幼师。。。你也要管，幼师的妹子好看呀！笨蛋

**Comment by zengzuo613** [comment]
汽车之家某部门，正招资深前端研发一名，负责前端工程化相关工作，职级和待遇可到阿里P7，工作强度不到985，工作地点北京-中关村，感兴趣的同学，请联系我！邮箱：zengzuo613@126.com

重点：【只看能力，不看年龄】

**Reply by isMrFan** [reply]
您好

**Reply by WangYingJay** [reply]
您好 方便了解一下吗？

**Reply by WangYingJay** [reply]
code_yingjie@163.com 这是我的邮件，随时找我，目前待业

**Reply by zengzuo613** [reply]
请通过邮箱，联系我

**Reply by tonyqus** [reply]
操。。。工作强度985。。。我还以为是学校985。。。滚~

**Comment by httaotao** [comment]
祝好呀，又一位优秀的大佬离开这个行业了，非常遗憾。作为后来的年轻人，看着各位朋友的现状，真的不知道未来路在何方。

**Comment by ghost** [comment]
开源框架作者转行卖钢材？？这跨度是不是有点忒大了，哈哈。

**Reply by ymbkxc** [reply]
可能之前公司是钢铁相关的业务

**Comment by alexinea** [comment]
加油，祝凯旋

**Comment by zgcwkj** [comment]
祝安好（一直在群😌

**Comment by CKernal** [comment]
听到这个消息确实很遗憾

**Comment by runME** [comment]
令人唏嘘，祝今后一切顺利！

**Comment by MicroHeartWangZheng** [comment]
有点感慨，祝作者以后顺利。

**Comment by JiyunDeng** [comment]
就不能转其他编程语言或岗位吗？转Java、go？

**Reply by httaotao** [reply]
都是半斤八两的，现在java和go随便找个实习生新人就能做了，很多公司也不会给机会了，hr都敢把你简历扔了，已经和语言无关了。

**Reply by XiaoNianPoPo** [reply]
缺乏核心竞争力唉。。。

**Reply by WheelerLee** [reply]
早些年行情好的时候，各行各业都纷纷培训加入了程序员大军

**Comment by JuninW** [comment]
祝未来一切顺利

**Comment by sankeyangshu** [comment]
祝未来一切顺利！

**Comment by yyancy** [comment]
祝好!!!!

**Comment by ByteRax** [comment]
祝未来一切顺利！
朋友，坚定地相信未来吧
相信不屈不挠的努力
相信战胜死亡的年轻
相信未来、热爱生命

**Comment by nano71** [comment]
我现在23岁, 但仿佛预知到自己十多年后的样子了, 祝好!

**Comment by leaf3woods** [comment]
毕业后最后悔的就是因为C++太难，跳到了C#岗位

**Comment by XiaoNianPoPo** [comment]
祝好

**Comment by leecus** [comment]
good luck

**Comment by shiguang2023** [comment]
国内的行情，对每个从事人员而言都是悲哀

**Comment by Singl-Daemon** [comment]
祝好

**Comment by mrdulin** [comment]
Respect

**Comment by Silence-dream** [comment]
向您致敬

**Comment by enochi** [comment]
加油,总会好起来的.

**Reply by chengshengguo** [reply]
祝好！

**Comment by starryknight1231** [comment]
祝好！ Respect

**Comment by csh0101** [comment]
祝好

**Comment by Jaylenwa** [comment]
good luck

**Comment by LeviTK** [comment]
祝你好运

**Comment by Tr33NewBee** [comment]
祝好，各位

**Comment by Mustenaka** [comment]
祝安好

**Comment by JaxnBai** [comment]
死皮赖脸地也要活下去，活下去才有希望
